templates/corporate/hello.html

```diff
@@ -1,562 +1,385 @@
+                                    </g>
+                                    <defs>
+                                        <clipPath id="clip0_765_19667">
+                                            <rect width="34" height="34" fill="white" />
+                                        </clipPath>
+                                    </defs>
+                                </svg>
+                            </div>
+                            With Zulip, questions or ideas were <strong>spread openly</strong>...
+                            <div class="quote__text-tail"></div>
+                        </div>
+                        <div class="quote__source">
+                            <a href="https://perso.univ-rennes1.fr/christophe.ritzenthaler/"><strong>Christophe Ritzenthaler</strong></a>,<i class="quote__source-title">Executive Director of CIMPA</i>
+                        </div>
+                    </div>
+                    <div class="quote">
+                        <div class="quote__text">
+                            <div class="quote__text-industry">
+                                <a href="/for/communities/">Communities</a>
+                                <svg class="quote__text-industry-icon _wider" viewBox="0 0 40 32" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                    <path d="M20 16.25C22.7167 16.25 25.1167 16.9 27.0667 17.75C28.8667 18.55 30 20.35 30 22.3V25H10V22.3167C10 20.35 11.1333 18.55 12.9333 17.7667C14.8833 16.9 17.2833 16.25 20 16.25ZM6.66667 16.6667C8.5 16.6667 10 15.1667 10 13.3333C10 11.5 8.5 10 6.66667 10C4.83333 10 3.33333 11.5 3.33333 13.3333C3.33333 15.1667 4.83333 16.6667 6.66667 16.6667ZM8.55 18.5C7.93333 18.4 7.31667 18.3333 6.66667 18.3333C5.01667 18.3333 3.45 18.6833 2.03333 19.3C0.8 19.8333 0 21.0333 0 22.3833V25H7.5V22.3167C7.5 20.9333 7.88333 19.6333 8.55 18.5ZM33.3333 16.6667C35.1667 16.6667 36.6667 15.1667 36.6667 13.3333C36.6667 11.5 35.1667 10 33.3333 10C31.5 10 30 11.5 30 13.3333C30 15.1667 31.5 16.6667 33.3333 16.6667ZM40 22.3833C40 21.0333 39.2 19.8333 37.9667 19.3C36.55 18.6833 34.9833 18.3333 33.3333 18.3333C32.6833 18.3333 32.0667 18.4 31.45 18.5C32.1167 19.6333 32.5 20.9333 32.5 22.3167V25H40V22.3833ZM20 5C22.7667 5 25 7.23333 25 10C25 12.7667 22.7667 15 20 15C17.2333 15 15 12.7667 15 10C15 7.23333 17.2333 5 20 5Z" fill="currentColor" />
+                                </svg>
```

Comment:
Should these SVGs be in the CSS file too?

Reply:
It should, just size wasn't standard. @amanagr could you put it in css similar to other?

Reply:
Done